### ISSUE TITLE
Attribute Error?

### ISSUE THREAD

**Issue by amcinto** [ildoonet/tf-pose-estimation#38]
I have had some issues with tensorflow and I was wondering if anyone has come across this error while using tensorflow 1.4.1(built from sources) and python3. [](url)https://github.com/ildoonet/tf-lcnn/issues
`AttributeError: 'LookupAlignConvolution2d' object has no attribute '_convolution_op'`

**Comment by amcinto** [comment]
I have ran slim.convolution and this works fine but when I call the lookup function I get this error. Is there any reason for this? The call to _convolution_op is there in the python distribution packages.

**Comment by amcinto** [comment]
https://github.com/ildoonet/tf-lcnn/issues

**Comment by ildoonet** [comment]
@amcinto Are you experiencing this issue after you update source codes with the latest?

**Comment by amcinto** [comment]
@ildoonet I do experience this error but I fixed the issue by adding a few lines of code to the LookupConvolution.py file. I did realize that I had to update the some  of the code because some of the code were updated by the developers. (e.g. tensorpack was updated when I installed and I had to switch the line of code from from `from tensorpack.dataflow.prefetch import PrefetchData` to `from tensorpack.dataflow import PrefetchData`.

**Comment by yw155** [comment]
Hi @amcinto , when you modified the codes to 'from tensorpack.dataflow import PrefetchData', do you meet the problem of “AttributeError: Can't pickle local object 'MapDataComponent.__init__.<locals>.f'” in line 368 of pose_dataset.py? Thanks.

**Comment by amcinto** [comment]
Hey @yw155, I am sorry but I posted this error for tf-lcnn. I ran into this error while I was trying to get the project up and running but I ended up fixing it by adding some code that may or may not have been missing. I was trying to reach out to @ildoonet but he got back to me on the other github page. This should still help though. I posted on the tensorpack github about tensorpack having an issue. I can post the link if you are interested. I'll close the issue though.
